docs/basic_usage.md

```diff
@@ -0,0 +1,62 @@
+# DAPper Basic Usage Guide
+
+DAPper (Dependency Analysis Project) is a tool that analyzes C/C++ source code to discover implicit dependencies and identify which system packages are required. This guide covers the basic usage patterns and common workflows.
+
+## Overview
+
+DAPper helps you:
+- **Discover header dependencies**: Find all `#include` statements in your C/C++ code
```

Comment:
I'd be inclined to rephrase this to discovering compile or build time dependencies based on header files -- probably with the the `map header files to the packages that provide them` added to the part here about finding all `#include` statements in your C/C++ code.